tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/Tests/GeneratorTests/test/test2.schema.dialog

```diff
@@ -0,0 +1,260 @@
+                }
+            }
+        },
+        "Ordinal": {
+            "type": "string",
+            "$mappings": [
+                "ordinal"
+            ]
+        },
+        "Location": {
+            "type": "object",
+            "$mappings": [
+                "geographyV2:Location",
+                "geographyV2"
+            ],
+            "required": [
+                "type",
+                "location"
+            ],
+            "properties": {
+                "type": {
+                    "type": "type"
+                },
+                "location": {
+                    "type": "location"
+                }
+            }
+        },
+        "KeyPhrase": {
+            "type": "string",
+            "$mappings": [
+                "keyPhrase"
+            ]
+        },
+        "Money": {
+            "type": "object",
+            "$mappings": [
+                "money:Money",
+                "money"
+            ],
+            "required": [
+                "number",
+                "units"
+            ],
+            "properties": {
+                "number": {
+                    "type": "number"
+                },
+                "units": {
+                    "type": "string"
+                }
+            }
+        },
+        "Percentage": {
+            "type": "string",
+            "$mappings": [
+                "percentage"
+            ]
+        },
+        "Temperature": {
+            "type": "object",
+            "$mappings": [
+                "temperature:Temperature",
+                "temperature"
+            ],
+            "required": [
+                "number",
+                "units"
+            ],
+            "properties": {
+                "number": {
+                    "type": "number"
+                },
+                "units": {
+                    "type": "string"
+                }
+            }
+        },
+        "Url": {
+            "type": "string",
+            "$mappings": [
+                "url"
+            ]
+        },
+        "Quantity": {
+            "type": "number",
+            "$mappings": [
+                "number"
+            ],
+            "minimum": 1,
+            "maximum": 10
+        },
+        "Directions": {
+            "type": "string",
+            "$mappings": [
+                "utterance"
+            ],
+            "$comment": "This indicates the whole string is what maps to this."
+        },
+        "Length": {
+            "type": "object",
+            "$mappings": [
+                "dimension:Length",
+                "dimension"
+            ],
+            "required": [
+                "number",
+                "units"
+            ],
+            "properties": {
+                "number": {
+                    "type": "number"
+                },
+                "units": {
+                    "type": "string"
+                }
+            }
+        },
+        "Bread": {
```

Comment:
>Bread [](start = 9, length = 5)

I go back and forth a bit from a unit test perspective on whether having bread/cheese or if we should instead have EnumeratedProperty1, EnumeratedProperty2 which makes it clearer what is being tested.